tests/playwright-test/reporter-html.spec.ts

```diff
@@ -2890,6 +2890,7 @@ for (const useIntermediateMergeReport of [true, false] as const) {
 
       await page.getByRole('link', { name: 'sample' }).click();
       await page.getByRole('button', { name: 'Copy prompt' }).click();
+      await page.waitForFunction(() => navigator.clipboard.readText());
```

Comment:
use the returned value to get the prompt text?

Reply:
It returns a JSHandle - not string so it makes things more complicated.

Reply:
`jsHandle.evaluate(...)`, but I don't feel too strongly about it.